_posts/en/newsletters/2024-10-11-newsletter.md

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Bitcoin Optech Newsletter #324'
+permalink: /en/newsletters/2024/10/11/
+name: 2024-10-11-newsletter
+slug: 2024-10-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- FIXME:harding btcd disclosure details set to be announced Thursday
+
+## FIXME instagibbs article
+
+FIXME:harding, see PR 1937
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[add getorphantxs][review club 30793] is a PR by [tdb3][gh tdb3] that
+adds a new experimental RPC method named `getorphantxs`. Since it is
```

Comment:
```suggestion
adds an experimental `getorphantxs` RPC. Since it is
```

Reply:
Leaving as-is.  I think both phrasings are fine and this isn't a section I wrote, so I'd prefer to let the original author's version stand.